### PR TITLE
adding issue template

### PR DESCRIPTION
**Pull request by djdanielsson**
# What does this PR do?

adding issue template

# How should this be tested?

none

# Is there a relevant Issue open for this?

no

# Other Relevant info, PRs, etc

none

### CODE REVIEW

**Comment by djdanielsson** [comment]
I also found some other stuff that looked like it was copied from controller config repo that still referenced CC so I updated that to be hub config, if for some reason it was correct I can change it back

File: .github/workflows/infra-release.yml [hunk 1]
```diff
@@ -54,15 +54,15 @@ jobs:
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: infra-controller_configuration-${{ github.ref_name }}.tar.gz
+          file: infra-cah_configuration-${{ github.ref_name }}.tar.gz
```
Comment (sean-m-sullivan) [inline .github/workflows/infra-release.yml]: cah?

Comment (djdanielsson) [reply .github/workflows/infra-release.yml]: yea you haven't heard of that?

**Review by sean-m-sullivan — APPROVED** [review]
LGTM

**Pull request merged**